portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,32 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <title>Guti's Portfolio</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="icon" type="image/ico" href="assets/images/favicon.ico"/>
+    <script src="assets/js/script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div id="header">
+        <h1>Guti's Portfolio</h1>
+        <a href="https://github.com/axgtz/">Github</a>
+        <a href="https://www.linkedin.com/in/rgtzgn/">linkedin</a>
+        <a href="https://www.instagram.com/rxgtz/">IG</a>
+    </div>
+    <div id="content" class="center">
+        <img id="avatar-image" src="assets/images/profile_pic.jpeg" />
+        <h1><div id="greeting-container">Welcome</div> to Guti's Portfolio</h1>
+        <p>My name is Alejandro Guti. Currently a senior Computer Science student from the ITESM in Mexico City. 
+            I'm a challenge driven person. </p>
+        
+        <h1> Hoobies </h1>
```

Comment:
Remove extra tags around values.  For example, I don't think is any difference between " Hobbies " and "Hobbies".